hal/src/boron/network/sara_ncp_client.h

```diff
@@ -95,10 +95,15 @@ class SaraNcpClient: public CellularNcpClient {
     system_tick_t regCheckTime_;
     system_tick_t registeredTime_;
     system_tick_t powerOnTime_;
+    unsigned int fwVersion_;
     bool memoryIssuePresent_ = false;
     unsigned registrationTimeout_;
     volatile bool inFlowControl_ = false;
 
+    const system_tick_t largePacketTimeoutMs_ = 250;
```

Comment:
Let's probably move this to the source file into the anonymous namespace constants.